Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -83,10 +95,13 @@ internal GoogleCredential(ICredential credential)
         /// </list>
         /// </summary>
         /// <returns>A task which completes with the application default credentials.</returns>
-        public static Task<GoogleCredential> GetApplicationDefaultAsync()
-        {
-            return defaultCredentialProvider.GetDefaultCredentialAsync();
-        }
+        public static async Task<GoogleCredential> GetApplicationDefaultAsync(IEnumerable<string> scopes) =>
```

Comment:
One downside of this is that if you call it multiple times, you'll get a different credential each time. That caused us problems a while ago in terms of creating too many channels.